### PR TITLE
Adopted vaccinespotter v0 'API'

### PR DESCRIPTION
**Pull request by seandcoburn**
Reworked the script to use the smaller file size of the 'api' vaccinespotter implemented, which is a simply .json file that gets updated at the same URL every minute or so.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,13 @@
...
 
-checks [COVID-19 Vaccine Spotter](https://www.vaccinespotter.org) for COVID-19 vaccine every 2 seconds
-
-beeps a bunch when it finds one
+Simple script to check [COVID-19 Vaccine Spotter](https://www.vaccinespotter.org) for COVID-19 vaccine appointments every one second and beeps a bunch when it finds one!
 
 ## prerequisites
 
-you need to have ruby
+You need to have ruby
```
Comment (getaaron) [inline README.md:7]: Nice

File: README.md [hunk 2]
```diff
@@ -1,15 +1,13 @@
...
 
- - [mac instructions](https://stackify.com/install-ruby-on-your-mac-everything-you-need-to-get-going/)
- - [windows instructions](https://stackify.com/install-ruby-on-windows-everything-you-need-to-get-going/)
+ - [macOS instructions](https://stackify.com/install-ruby-on-your-mac-everything-you-need-to-get-going/)
```
Comment (getaaron) [inline README.md:9]: Love it CC @robmadden

**Review by getaaron — APPROVED** [review]
LGTM

**Comment by getaaron** [comment]
@mgates please provide positive & specific feedback

**Comment by robmadden** [comment]
tight

**Comment by robmadden** [comment]
tight ... no linux instrux tho?

**Comment by robmadden** [comment]
https://www.youtube.com/watch?v=wsXhk1xmYAI

**Comment by seandcoburn** [comment]
> tight ... no linux instrux tho?
> […](#)

linstructions coming in next update.

**Comment by getaaron** [comment]
Are there a lot of people who have Linux but need help installing Ruby? 😀

**Comment by robmadden** [comment]
Good point @getaaron. Nonetheless, if I were still a man of principle and thereby a Linux user it'd be rad if there were instructions tailored towards Linux users.

https://www.youtube.com/watch?v=DB6eg4Esggg

**Comment by robmadden** [comment]
Also think about how much typing you'll be saving.

**Pull request merged**